lua/telescope/make_entry.lua

```diff
@@ -431,17 +431,27 @@ function make_entry.gen_from_git_commits(opts)
       return nil
     end
 
-    local sha, msg = string.match(entry, "([^ ]+) (.+)")
+    local marker, sha, msg = string.match(entry, "([*\\/| ]+) +([0-9a-f]*) +(.*)")
+
+    if not sha then
+      marker = entry
+      sha = ""
+      msg = ""
+    end
 
     if not msg then
-      sha = entry
       msg = "<empty commit message>"
     end
```

Comment:
won't `not msg` be always false?

Reply:
i think the check makes sense. its nil (and we enter this branch) if we match a line that has no commit message (its nil because of string.match) but its and empty string if we dont even have a sha, so we do not enter this branch